### ISSUE TITLE
The installer stops reporting "fdisk -l on /dev/mmcblk0rpmb" returned an error

### ISSUE THREAD

**Issue by lucabelluccini** [bryteise/ister#157]
In some boards (e.g. [UDOO X86](https://www.udoo.org/udoo-x86/)) equipped with EMMC, the block devices can be the following:
```
/dev/mmcblk0
/dev/mmcblk0boot0
/dev/mmcblk0boot1
/dev/mmcblk0p1
/dev/mmcblk0p2
/dev/mmcblk0p3
/dev/mmcblk0rpmb
```

The `fdisk -l` will fail on `/dev/mmcblk0rpmb`:
```
udoox86 ~ $ sudo fdisk -l /dev/mmcblk0rpmb
fdisk: cannot open /dev/mmcblk0rpmb: Illegal seek
```

More info about `mmcblk0rpmb` can be found at [LWN](https://lwn.net/Articles/682276/).

The installer (e.g. used in Clear Linux) fail with no way to recover in case those devices are scanned.

I have a patch to skip it. Would you prefer filtering it at `get_list_of_disks()` or in `get_disk_info()`?

**Comment by lucabelluccini** [comment]
Hello @bryteise, I'll run a test this weekend to understand why the `fdisk -l` has a try-except but the exception somehow passes through.

**Comment by bryteise** [comment]
Ah well I'll keep the patch around for awhile in the code base then since it isn't hurting. No need to have a more comprehensive fix beyond #158 at least. Thanks for the attention to the issue.
